### PR TITLE
Lrsd 4177

### PR DESCRIPTION
**Pull request by amosfong**

### CODE REVIEW

**Comment by amosfong** [comment]
ci:forward:force

**Comment by amosfong** [comment]
ci:test:sf

File: workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/client-extension.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 assemble:
     - from: build/vite
-      into: static
+        into: static
```
Comment (ryanschuhler) [inline workspaces/liferay-customer-workspace/client-extensions/liferay-customer-custom-element/client-extension.yaml:3]: @amosfong this extra two spaces seems to break the deploy

File: workspaces/liferay-customer-workspace/client-extensions/liferay-customer-global-css/client-extension.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 assemble:
     - from: assets
-      into: static
+        into: static
```
Comment (ryanschuhler) [inline workspaces/liferay-customer-workspace/client-extensions/liferay-customer-global-css/client-extension.yaml:3]: @amosfong this extra two spaces seems to break the deploy

Comment (amosfong) [reply workspaces/liferay-customer-workspace/client-extensions/liferay-customer-global-css/client-extension.yaml:3]: oh my formatsource did that, I'll revert it

**Comment by amosfong** [comment]
ci:test:sf

**Comment by ryanschuhler** [comment]
closing for https://github.com/liferay-customer-solutions/liferay-portal/pull/223

**Pull request closed without merging**